nifi-framework-bundle/nifi-framework/nifi-framework-components/src/main/java/org/apache/nifi/reporting/AbstractEventAccess.java

```diff
@@ -540,6 +541,18 @@ ProcessGroupStatus getGroupStatus(final ProcessGroup group, final RepositoryStat
 
         final VersionControlInformation vci = group.getVersionControlInformation();
         if (vci != null) {
+            final RegisteredFlowSnapshotMetadata registeredFlowSnapshotMetadata = new RegisteredFlowSnapshotMetadata();
+            registeredFlowSnapshotMetadata.setBranch(vci.getBranch());
+            registeredFlowSnapshotMetadata.setBucketIdentifier(vci.getBucketIdentifier());
+            registeredFlowSnapshotMetadata.setFlowIdentifier(vci.getFlowIdentifier());
+            registeredFlowSnapshotMetadata.setVersion(vci.getVersion());
+            registeredFlowSnapshotMetadata.setComments(Map.of(
+                    "registryName", vci.getRegistryName(),
+                    "flowName", vci.getFlowName(),
+                    "flowDescription", vci.getFlowDescription(),
+                    "bucketName", vci.getBucketName(),
+                    "status", vci.getStatus().getState().name()).toString());
```

Comment:
Relying on the `Map.toString()` behavior is questionable here, along with including those other fields. It seems better to leave this blank, or limit the content to a single field like Flow Description.